torchao/prototype/scaled_grouped_mm/kernels/jagged_float8_scales.py

```diff
@@ -0,0 +1,356 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Triton kernels for scaling high precision tensors to float8 using "jagged"
+rowwise scales (i.e., separate scales for each group/subtensor as determined by
+the offsets).
+"""
+
+from typing import Tuple
+
+import torch
+import triton
+import triton.language as tl
+
+from torchao.prototype.scaled_grouped_mm.utils import _is_column_major
+
+EPS = 1e-12
+
+FP8_DTYPE_MAP = {
```

Comment:
Nit: I think there is a built in 1

Reply:
I found [this](https://github.com/triton-lang/triton/blob/981e987eed9053b952f81153bc0779c99d8c642e/python/triton/runtime/jit.py#L448), is this what you're referring to? Since it's just dtype string/name -> dtype string/name map there would be some extra steps to use it, not sure if it's worth it

Reply:
yeah no worries, leave as is